apps/hash-frontend/src/pages/[shortname]/[page-slug].page/canvas-page.tsx

```diff
@@ -121,7 +124,12 @@ export const CanvasPageBlock = ({
   };
 
   return (
-    <div style={{ height: "100%", width: "100%", position: "absolute" }}>
```

Comment:
@CiaranMn any particular reason I may have missed why this was set to `position: "absolute"`? Have removed this and doesn't seem to have had any unintended consequences as far as I can tell.

Reply:
Not sure.. it might have been leftover from a different implementation